spec/views/films/index.html.html_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'films/index.html.haml', type: :view do
+  it 'displays the title' do
+    assign(:films, [
+             { 'show_id' => 's1',
+               'type' => 'Movie',
+               'title' => 'Dick Johnson Is Dead',
+               'director' => 'Kirsten Johnson',
+               'cast' => nil,
+               'country' => 'United States',
+               'date_added' => 'September 25, 2021',
+               'release_year' => '2020',
+               'rating' => 'PG-13',
+               'duration' => '90 min',
+               'listed_in' => 'Documentaries',
+               'description' =>
+                 'As her father nears the end of his life, ...' }
+           ])
+
+    render
+
+    expect(rendered).to have_text('Smartflix')
```

Comment:
can you also verify that this movie is showing? So something like `expect(rendered).to have_text('Dick Johnson Is Dead')` Although since in the .csv file, you can change the assigned list to contain 1 object which has `title => 'Test title'` and then  do ``expect(rendered).to have_text('Test title')``

Reply:
@Goran1708 Sorry bout that, I was not very careful